extra/mender-client-docker-addons/Dockerfile

```diff
@@ -8,13 +8,21 @@ RUN apt-get update && apt-get install -y make git build-essential golang liblzma
 
 ARG MENDER_CLIENT_REV=master
 ARG MENDER_CONNECT_REV=master
+ARG MENDER_SETUP_REV=master
+ARG MENDER_SNAPSHOT_REV=master
 
 RUN git clone https://github.com/mendersoftware/mender /src/mender
 RUN (cd /src/mender && git fetch origin $MENDER_CLIENT_REV && git checkout FETCH_HEAD || git checkout -f $MENDER_CLIENT_REV)
 
 RUN git clone https://github.com/mendersoftware/mender-connect /src/mender-connect
 RUN (cd /src/mender-connect && git fetch origin $MENDER_CONNECT_REV && git checkout FETCH_HEAD || git checkout -f $MENDER_CONNECT_REV)
 
+RUN git clone https://github.com/mendersoftware/mender-setup /src/mender-setup
+RUN (cd /src/mender-setup && git fetch origin $MENDER_SETUP_REV && git checkout FETCH_HEAD || git checkout -f $MENDER_SETUP_REV)
+
+RUN git clone https://github.com/mendersoftware/mender-snapshot /src/mender-snapshot
+RUN (cd /src/mender-snapshot && git fetch origin $MENDER_SNAPSHOT_REV && git checkout FETCH_HEAD || git checkout -f $MENDER_SNAPSHOT_REV)
```

Comment:
You need to install it too, no?

Also I think you can omit mender-snapshot, it's unusable from this container.

Reply:
You are right. Removing